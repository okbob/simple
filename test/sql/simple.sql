CREATE EXTENSION simple;

SELECT int_func(10);
SELECT text_func('Ahoj');
