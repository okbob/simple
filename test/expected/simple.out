CREATE EXTENSION simple;
SELECT int_func(10);
 int_func 
----------
       20
(1 row)

SELECT text_func('Ahoj');
NOTICE:  input string is: "Ahoj"
  text_func  
-------------
 Ahoj, svÄ›te
(1 row)

